tests/commands/test_init.py

```diff
@@ -63,17 +65,43 @@ def test_all_the_files(tmp_path):
 def test_bad_name(tmp_path):
     cmd = InitCommand('group')
     with pytest.raises(CommandError):
-        cmd.run(Namespace(path=tmp_path, name='1234', author="שראלה ישראל"))
+        cmd.run(Namespace(path=tmp_path, name='1234', author="שראלה ישראל", series=[]))
 
 
 def test_executables(tmp_path):
     cmd = InitCommand('group')
-    cmd.run(Namespace(path=tmp_path, name='my-charm', author="홍길동"))
+    cmd.run(Namespace(path=tmp_path, name='my-charm', author="홍길동", series=[]))
     assert (tmp_path / "run_tests").stat().st_mode & S_IXALL == S_IXALL
     assert (tmp_path / "src/charm.py").stat().st_mode & S_IXALL == S_IXALL
 
 
 def test_tests(tmp_path):
     cmd = InitCommand('group')
-    cmd.run(Namespace(path=tmp_path, name='my-charm', author="だれだれ"))
+    cmd.run(Namespace(path=tmp_path, name='my-charm', author="だれだれ", series=[]))
     subprocess.run(["./run_tests"], cwd=str(tmp_path), check=True)
+
+
+def _test_metadata_series(tmp_path, *, os_release=None, args=(), expected):
+    with patch('charmcraft.commands.init.parse_os_release') as osr:
+        osr.return_value = os_release
+
+        cmd = InitCommand('group')
+        cmd.run(Namespace(path=tmp_path, name='my-charm', author="fred", series=args))
+        subprocess.run(["./run_tests"], cwd=str(tmp_path), check=True)
```

Comment:
Why are you running all the tests again here?

Reply:
by mistake

Reply:
Given path=tmp_path this is saying we are running the test suite *for the initialized charm*, not the test suite for charmcraft.

Reply:
yes, this was because i started from the test that runs the unit tests of the template charm :-)